include/json/reader.h

```diff
@@ -135,7 +135,9 @@ class JSONCPP_DEPRECATED(
    * \param message The error message.
    * \return \c true if the error was successfully added, \c false if the Value
    * offset exceeds the document size.
+   * \deprecated Not suggest for use.
    */
+  JSONCPP_DEPRECATED("Not suggest for use.")
```

Comment:
Isn't ALL of Reader deprecated? Why does this need to be deprecated too?

If it does, can you say in the deprecation message what you are supposed to do instead?